tensorflow/lite/g3doc/convert/index.md

```diff
@@ -50,6 +50,7 @@ In case you encounter any issues:
     (from the [tflite_convert](https://www.tensorflow.org/lite/convert/cmdline)
     command line tool) or `converter.experimental_new_converter=False` (from the
     [Python API](https://www.tensorflow.org/api_docs/python/tf/lite/TFLiteConverter))
+*   If you want to convert a Frozen Graph you can use the `from_frozen_graph()` method from `tf.compat.v1.lite.TFLiteConverter.from_frozen_graph()`. One advantage of using it is it allows us to specify `input_arrays` and `output_arrays` that would go into the converted TensorFlow Lite model.   
```

Comment:
Can you point to the section on 1.x compatibility doc? Please use relative path.
https://www.tensorflow.org/lite/convert/1x_compatibility#frozen_graph

Besides, removing input_arrays and output_arrays is a deliberate choice to align with SavedModel format in TF 2 so specifying `input_arrays` and `output_arrays` shouldn't be considered an advantage but a backward compatible mode.

cc: @jdduke

Reply:
I did not realize this stuff is already specified in [this guide](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/convert/1x_compatibility.md). So, I think it's probably better that I close the PR.